.travis.yml

```diff
@@ -211,3 +211,7 @@ before_script:
 script:
   # Pick one test in this directory and verify
   - time ./toolset/run-ci.py verify "$TESTDIR"
+
+cache:
+  directories:
+  - $HOME/.m2/repository
```

Comment:
This is interesting, but not the correct directory. `mvn` (and others) are run by the testrunner user, so it needs to be `/home/testrunner/.m2`, I believe.